components/expenses/__tests__/Expenses.test.js

```diff
@@ -80,7 +80,7 @@ describe('Expenses component', () => {
       expect(component.find('[data-cy="mark-expense-as-paid-btn"]')).not.toBeFalsy();
 
       // make sure none are disabled
-      expect(component.find('[data-cy="expense-actions"] button[disabled]').lenght).toEqual(undefined);
+      expect(component.find('[data-cy="expense-actions"] button[disabled]').length).toEqual(4);
```

Comment:
Looks like this is not matching the comment. Not sure what to do.

Reply:
Maybe the code is incorrect?

Reply:
I'm tempted to discard this line, it's better to have nothing than a false assumption. Probably not worth digging.

Reply:
I will remove it :+1: